Library/Homebrew/cask/audit.rb

```diff
@@ -768,13 +768,15 @@ def audit_livecheck_version
     sig { void }
     def audit_min_os
       return unless online?
-      return unless strict?
 
       odebug "Auditing minimum macOS version"
 
       bundle_min_os = cask_bundle_min_os
       sparkle_min_os = cask_sparkle_min_os
 
+      puts "Bundle min os:", bundle_min_os
+      puts "Sparkle min os:", sparkle_min_os
+
```

Comment:
```suggestion
```

Reply:
Sorry about that! Thanks for picking it up.